corehq/apps/custom_data_fields/migrations/0011_alter_customdatafieldsdefinition_profile_required_for_user_type.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.15 on 2024-10-30 14:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Shouldn't this just be the 0010 migration?

Reply:
This is a question I was wrestling with. It was a mess to undo the original migration locally and it seemed like it would be inadvisable to try to undo the original 0010 migration on staging. Having 0011 on top of this is the path of least resistance, but that doesn't mean it's the best path. @esoergel Do you have any thoughts on this. The basic issue is that the original 0010 migration added a charfield to the `CustomDataFieldsDefinition` and was deployed to staging. That was then updated to a json field with 0011. I was anticipating issues if I were to just delete the original 0010 migration and replace it with a new 0010 that applied the json field.

Reply:
fixed here https://github.com/dimagi/commcare-hq/pull/35266/commits/80c3f5b120b7c459b7f41a865487cbbbd9bebb92
didn't have the same issue with staging as I did locally